lib/libc/sys/syscalls.S

```diff
@@ -1,5 +1,7 @@
 #include <machine/syscall.h>
 
+.cfi_sections .debug_frame, .eh_frame
```

Comment:
I removed those entirely, rebuild repo from scratch and kernel-user transition in backtrace still works. Are those really needed? What they were supposed to do then?

Reply:
According to [doc](https://sourceware.org/binutils/docs/as/CFI-directives.html#CFI-directives):

> `.cfi_sections` may be used to specify whether CFI directives should emit `.eh_frame` section and/or `.debug_frame` section. If section_list is `.eh_frame`, `.eh_frame` is emitted, if section_list is `.debug_frame`, `.debug_frame` is emitted. To emit both use `.eh_frame`, `.debug_frame`. 

But:
> The default if this directive is not used is .cfi_sections .eh_frame.

So we should be fine without these.